src/components/button/button.tsx

```diff
@@ -1,13 +1,36 @@
-import React, { FC } from 'react';
+import React, { FC, ReactNode } from 'react';
+import classnames from 'classnames';
 
 export interface ButtonProps {
-	id: string;
+	label?: string;
+	onClick: () => void;
+	color: 'base' | 'primary' | 'secondary';
+	size: 'm' | 'l';
+	icon?: ReactNode;
```

Comment:
The icon could maybe be solved with as a component type as it was done in the input component. Therefore we can tell what Icon we want. Therefore we have the restriction to also set the props of a icon and prevent the user for example from setting the icon size as "m" accidentally.

Reply:
@malinovic fixed